test/cypress/tests/ui/DataEmpty.cy.ts

```diff
@@ -0,0 +1,71 @@
+import { createEditorWithTextBlocks } from '../../support/utils/createEditorWithTextBlocks';
+
+describe('inputs [data-empty] mark', function () {
+  it('should be added to the editor on initialization', function () {
```

Comment:
```suggestion
  it('should be added to blocks on the editor initialization', function () {
```

Reply:
The test case name sounds like this attribute should be added to the editor container.

Reply:
fixed